package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "start": "cross-env NODE_ENV=production node src/index.js",
     "start:dev": "cross-env NODE_ENV=development nodemon src/index.js",
+    "run-seeds": "cd src/ & npx sequelize-cli db:migrate:undo:all && npx sequelize-cli db:migrate && npx sequelize-cli db:seed:all",
```

Comment:
That's what I asked, great, thanks. 
But please rename this script into `db-migration`

Reply:
Renamed ✔️